src/lib.rs

```diff
@@ -113,6 +113,17 @@ mod tests {
 
         let decrypted = decrypt_any(&encrypted, amount);
 
+        assert_eq!(plain, decrypted);
+    }
+    #[test]
+    fn test_rot13_abc() {
```

Comment:
We already have a rot13 example?

Reply:
Actually... Meh whatever it can stay

Reply:
Well sure, it wasn't meant as an example, it's the missing test case that would have caught the bug!
So, finally, the crypto library should have (near) complete test coverage :-)